Sources/ArcGISToolkit/Extensions/ArcGIS/AttachmentHelpers/FeatureAttachment.swift

```diff
@@ -48,8 +45,8 @@ public protocol FeatureAttachment: Loadable {
     /// The name of the attachment.
     var name: String { get }
     
-    /// The size of the attachment in bytes.
-    var size: Int { get }
+    /// The size of the attachment.
+    var measuredSize: Measurement<UnitInformationStorage> { get }
```

Comment:
This was cleaner than completely dropping `size` from the protocol.

Dropping it completely meant we'd need to add checking for `PopupAttachments`/`FormAttachments` in `AttachmentList` and `AttachmentPreview`.